### PR TITLE
Add Aero Porter

### PR DESCRIPTION
**Pull request by jdenm8**
Adds Aero Porter.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by jdenm8** [comment]
The form requested using the homescreen for the boxart, but this game's homescreen really sucks. Maybe use the logo should be used instead?

**Comment by jdenm8** [comment]
Pre-emptively fixed capitalisation of folder name.

File: Aero-Porter/game.dat [hunk 1]
```diff
@@ -0,0 +1,25 @@
+title = "Aero Porter"
+description = "Aero Porter is a Simulation Game developed by Vivarium Inc. and published by Level-5.\n\nAero Porter was originally released at Retail as part of the Guild01 compilation in Japan. This page concerns itself with the separate eShop release. For information on the Japanese Retail release it was a part of, please see the page for Guild01."
```
Comment (TheKoopaKingdom) [inline Aero-Porter/game.dat]: The newline escape codes (`\n`) aren't needed.

**Comment by TheKoopaKingdom** [comment]
>The form requested using the homescreen for the boxart, but this game's homescreen kinda sucks. Maybe the logo should be used instead?

I think it's fine.

>I also couldn't find the Title ID for the Japanese eShop release.

That's alright.

**Comment by TheKoopaKingdom** [comment]
LGTM.

**Pull request merged**